modules/bot-improvement/src/views/full/index.tsx

```diff
@@ -0,0 +1,199 @@
+import { Callout, Icon } from '@blueprintjs/core'
+import { Tab, Tabs } from '@blueprintjs/core'
+import { Container } from 'botpress/ui'
+import _ from 'lodash'
+import React, { FC, useEffect, useState } from 'react'
+
+import { FeedbackItem, Goal, QnAItem } from '../../backend/typings'
+
+import { makeApi } from './api'
+import Conversation from './components/messages/Conversation'
+import FeedbackItemComponent from './components/FeedbackItem'
+import style from './style.scss'
+
+type SelectedTabId = 'pending' | 'solved'
+
+const defaultState: {
+  feedbackItems: FeedbackItem[]
+  qnaItems: QnAItem[]
+  goals: Goal[]
+  feedbackItemsLoading: boolean
+  currentFeedbackItem: FeedbackItem
+  selectedTab: SelectedTabId
+} = {
+  feedbackItems: [],
+  qnaItems: [],
+  goals: [],
+  feedbackItemsLoading: true,
+  currentFeedbackItem: null,
+  selectedTab: 'pending'
+}
+
+const getDefaultQnaItemId = (qnaItems: QnAItem[]) => {
+  return qnaItems[0].id
+}
+
+const getDefaultGoalId = (goals: Goal[]) => {
+  return goals[0].id
+}
```

Comment:
Since they are helper methods, shouldn't you check the array length to ensure there's at least one element, or return an empty field?